### ISSUE TITLE
Possible adjoint strain seismograms mix-up

### ISSUE THREAD

**Issue by eyalshimony** [geodynamics/specfem3d#1432]
In the following code, which, as far as I understand, is responsible for creating the channel part of the file name of the adjoint strain seismograms (in the write_seismograms file, write_adj_seismograms2_to_file subroutine, lines 898-913):
``` 
if (idimval == 1 .and. jdimval == 1) then
    chn = 'SNN'
else if (idimval == 1 .and. jdimval == 2) then
    chn = 'SEN'
else if (idimval == 1 .and. jdimval == 3) then
    chn = 'SEZ'
else if (idimval == 2 .and. jdimval == 2) then
    chn = 'SEE'
else if (idimval == 2 .and. jdimval == 3) then
    chn = 'SNZ'
else if (idimval == 3 .and. jdimval == 3) then
    chn = 'SZZ'
```
there apparently seems to be a mix-up in the file names between the SNN file and the SEE file, from looking in the other file names, if I indeed understand the naming convention.

**Comment by danielpeter** [comment]
yes, you're right!  SPECFEM reference frame convention is: x = East, y = North, z = Up

that makes strain component eps_s(1,1) = eps_xx point in EE-direction and eps_s(2,2) = eps_yy in NN-direction. will be fixed.

note that we denote seismograms with N/E/Z only if a UTM conversion is used. in case the `SUPPRESS_UTM_PROJECTION` flag is set to `.true.`, we will denote this according to the Cartesian direction X/Y/Z directly. i will apply the same convention to these strain seismograms, thus depending on the UTM setting, it will be either `SEE` or `SXX`.
